custom/enikshay/tests/utils.py

```diff
@@ -27,6 +27,115 @@
 from corehq.apps.users.models import CommCareUser
 
 
+def get_person_case_structure(case_id, user_id, extra_update={}):
```

Comment:
it should be, though is more future proof to make the default `None` and then have `extra_update = extra_update or {}` at the top of the function. just in case later someone else updates this function to modify it some how